challenges-webapp/src/main/webapp/vue-app/challenges/components/ChallengeCard.vue

```diff
@@ -154,7 +156,7 @@ export default {
       }
     },
     enableAnnounce(){
-      return this.challenge && this.challenge.program  && this.challenge.userInfo.canAnnounce && this.status !== 'Ended' && this.status !== 'Starts';
+      return this.challenge && this.challenge.program && this.challenge.program.enabled && !this.challenge.program.deleted  && this.challenge.userInfo.canAnnounce && this.status !== 'Ended' && this.status !== 'Starts';
```

Comment:
We may use a computed for 
`this.challenge && this.challenge.program && this.challenge.program.enabled && !this.challenge.program.deleted `

Reply:
done!